internal/driver/glfw/window_test.go

```diff
@@ -1942,3 +1949,27 @@ func (t *tabbable) AcceptsTab() bool {
 	t.acceptTabCallCount++
 	return true
 }
+
+func safeResize(w fyne.Window, size fyne.Size) {
+	runOnMain(func() {
+		w.Resize(size)
+	})
+}
```

Comment:
I wonder if it perhaps would be easier, and less chance of new tests here not using these, if we had a wrapper around the window? We could extend it like we usually do with widgets basically and then have all of the glfw tests use that specific glfwTestWindow wrapper? Just an idea :)

Reply:
Absolutely. I'll open an issue for it

Reply:
https://github.com/fyne-io/fyne/issues/5375